packages/html/__tests__/HtmlImageLayer.test.ts

```diff
@@ -51,4 +53,55 @@ describe('HtmlImageLayer tests', function () {
         await flushPromises();
         expect(img.src).toEqualAnalyticsToken('AXAABABD');
     });
+
+    it('should set image src twice if HtmlImageLayer instance won\'t be destroyed', async function () {
+        const img = document.createElement('img');
+        const pluginsState: any = {
+            cleanupCallbacks: []
+        };
+        const spy = jest.spyOn(img, 'setAttribute');
+        const dummyFailingPlugin = (): Promise<PluginResponse> => {
+            return new Promise((resolve) => {
```

Comment:
This could be simplified with implicit arrow function returns (and the other plugin below too).